module/documents/actor.mjs

```diff
@@ -201,25 +412,131 @@ export default class ActorEd extends Actor {
    * <ul>
    *     <li>taking strain damage</li>
    *     <li>reducing resources (karma, devotion)</li>
+   *     <li>recover from damage</li>
    * </ul>
    * @param {EdRoll} roll The prepared Roll.
+   * @param {string} recoveryMode type of recovery
    */
-  #processRoll( roll ) {
+  #processRoll( roll, recoveryMode ) {
```

Comment:
This is way to specific a parameter for a function that should be generalized.
We need to find a way of processing this another way

Reply:
it would be a sub set of a subset:

testType = effect
rollType = recovery
recoveryMode = stun, standard, nightrest

check together

Reply:
subset ist richtig